src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -15,19 +16,24 @@ public static long countWords() throws IOException {
 				.get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
 		List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-		long count = 0;
-		for (String w : words) {
-		  if (w.length() > 12) count++;  
-		}
+		long count = words.stream()
+				.filter(w -> w.length() > 12)
+				.count();
 		return count;
```

Comment:
아래와 같이 바로 return해도 좋을 것 같네요 :)

```suggestion
 		return words.stream()
				.filter(w -> w.length() > 12)
				.count();
```

Reply:
네~ 리뷰 감사합니다. :)